### DISCUSSION TITLE
Payout api missing

### DISCUSSION THREAD

**Discussion by oiiahchai** [btcpayserver/btcpayserver#2141] [General]
Hello, been using the greenfield api for invoice and pull payment, however I can't found the payout api, I have to login to btcpay server to confirm the pull payment payout, i suggest payout api with webhook event.
Tq

**Comment by Kukks** [comment]
> Hello, been using the greenfield api for invoice and pull payment, however I can't found the payout api, I have to login to btcpay server to confirm the pull payment payout, i suggest payout api with webhook event.
> Tq

You can approve a payout via the API, although, we still have not implemented automatic transfers once a payout is approved and a hot wallet is available. (#1851 related)
![image](https://user-images.githubusercontent.com/1818366/102337967-77591000-3f93-11eb-98a8-b6e93f0fbe15.png)

**Comment by oiiahchai** [comment]
Hello, yes now I use this API to approve the payout, i actually looking for automatic transfer payout. Since there are approve payout api why not transfer the fund that approved via api ? security related ? how if provide option to store owner to enable/disable automatic fund transfer.
It not convenient since the payout is approved via api and have to login to btcpay server to confirm the payout.

**Reply by Kukks** [reply]
It's planned but not there yet.
